output/badger.go

```diff
@@ -0,0 +1,68 @@
+package output
+
+import (
+	"errors"
+	"fmt"
+	"github.com/dgraph-io/badger"
+	"github.com/rapid7/godap/api"
+	"github.com/rapid7/godap/factory"
+	"github.com/rapid7/godap/util"
+	"strings"
+)
+
+type OutputBadger struct {
+	db             *badger.DB
+	batch          *badger.WriteBatch
+	string_builder *strings.Builder
```

Comment:
Where is this being used to build the keys?

Reply:
Ah, good catch! It's actually unused now, since field_join now does what the initial prototype did. PR updated.